test/src/test_utils.c

```diff
@@ -75,6 +78,44 @@ ts_test_error_injection(PG_FUNCTION_ARGS)
 	PG_RETURN_VOID();
 }
 
+static int
+throw_every_nth_row(int max_rows, int severity)
```

Comment:
I am a little confused about the semantics here. It seems to me that it errors out _after_ N rows rather then every nth row. I mean, once it errors the first time it won't continue.

Reply:
Yep, makes sense, I should call it `throw_after_n_rows`.

Reply:
It also reads as if literally throw each nth row :black_cat:

Reply:
> It also reads as if literally throw each nth row

You mean, the new name `throw_after_n_rows` as well? I'm out of ideas, but if you suggest something, can rename.